frontend/src/app/components/forms/form-create/form-create.component.ts

```diff
@@ -0,0 +1,122 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {Form} from '../../../models/form.model';
+import {Location} from '@angular/common';
+import {FormSection} from '../../../models/form.section.model';
+import {AppState} from '../../../store/store.module';
+import {Store} from '@ngrx/store';
+import {FormUploadAction, FormUploadPublishAction, FullyLoadFormAction} from '../../../store/form/form.actions';
+import {Subscription} from 'rxjs';
+import {cloneDeep} from 'lodash';
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+
+@Component({
+  selector: 'app-form-create',
+  templateUrl: './form-create.component.html',
+  styleUrls: ['./form-create.component.scss']
+})
+export class FormCreateComponent implements OnInit, OnDestroy {
+
+  readonly FORM_ID_URL_PARAM = 'formId';
+
+  title: string;
+  form: Form;
+
+  showSection: boolean;
+  showOptions: boolean;
+
+  formDataSubscription: Subscription;
+
+  constructor(private location: Location,
+              private store: Store<AppState>,
+              private activatedRoute: ActivatedRoute) { }
+
+  ngOnInit() {
```

Comment:
I saw that the other forms in the project extend from `FormGroup` and use the new reactive forms approach instead of binding inputs to ngModel. Could you use the same approach here? (take a look at [observer profile](https://github.com/code4romania/monitorizare-vot-ong/tree/76664ceb45dfef0525b90005d79415a428756b98/frontend/src/app/components/observers/observer-profile))